Sources/Bookmarks/BookmarkUtils.swift

```diff
@@ -174,6 +174,16 @@ public struct BookmarkUtils {
         return (try? context.fetch(request)) ?? []
     }
 
+    public static func fetchTitlesForBookmarks(modifiedBefore date: Date, in context: NSManagedObjectContext) -> [String] {
+        let request = NSFetchRequest<NSFetchRequestResult>(entityName: "BookmarkEntity")
```

Comment:
You can use `BookmarkEntity.fetchRequest()` to avoid hardcoded strings.

Reply:
Yep, I saw it but that one is for `BookmarkEntity` result type, and for dictionary result type we need `NSFetchRequestResult`.